packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -34,13 +34,6 @@
           </template>
           {{ t('runs.connect.modal.createOrg.waitingButton') }}
         </Button>
-        <Button
```

Comment:
Is there any benefit to keeping this button around to just refetch the query (rather than poll) just incase there's something that goes wrong with the subscription/notification?